mcs/build/library.make

```diff
@@ -44,6 +44,12 @@ RESOURCE_STRINGS_FILES += $(RESOURCE_STRINGS:%=--resourcestrings:%)
 endif
 endif
 
+ifdef LIBRARY_WARN_AS_ERROR
```

Comment:
This shouldn't be done in the basic profile, which uses system mcs/csc which we don't control.